native_package_install.py

```diff
@@ -141,29 +141,40 @@ def get_source_dirs(vendor_dir, package):
     return data['source-directories']
 
 
+def replace_in_file(filePath, src, target):
+    """ find replace in a file """
+    lines = []
+    with open(filePath) as infile:
+        for line in infile:
+            line = line.replace(src, target)
+            lines.append(line)
```

Comment:
you can just do:

```
output = ""
with open(file_path) as in_file:
  output = in_file.read().replace(src, target)

with open(file_path, 'w') as out_file:
  out_file.write(output)
```

Reading line-by-line is only really needed for giant files. Here, we don't care because the size is never going to be big enough